demo/site/src/app/block-preview/page/page.tsx

```diff
@@ -34,11 +35,13 @@ const PreviewPage: React.FunctionComponent = () => {
 };
 const IFrameBridgePreviewPage = (): JSX.Element => {
     return (
-        <IFrameBridgeProvider>
-            <BlockPreviewProvider>
-                <PreviewPage />
-            </BlockPreviewProvider>
-        </IFrameBridgeProvider>
+        <IntlProvider locale="en" messages={{}}>
```

Comment:
This should be fixed differently

Reply:
I removed the intl dependency again. The aria label of the video preview button can be added later, or if needed the component can be implemented in the projects.